docs/guides/jest.md

```diff
@@ -22,7 +22,9 @@ several modules to be unmocked in your `package.json`:
     "object.assign",
     "define-properties",
     "function-bind",
-    "object-keys"
+    "object-keys",
+    "object.values",
```

Comment:
@ljharb we don't use `object.entries` though... why would we need to add it to this doc?